test/peds_test/peds_test.py

```diff
@@ -0,0 +1,532 @@
+
```

Comment:
your test should call `peds.py`, not duplicate it.

Reply:
The test file is updated and has passed on CircleCI.